docker/musl.sh

```diff
@@ -36,13 +36,33 @@ main() {
     # Don't depend on the mirrors of sabotage linux that musl-cross-make uses.
     local linux_headers_site=https://ci-mirrors.rust-lang.org/rustc/sabotage-linux-tarballs
 
+    # we need -fPIC for CFLAGS because old binutils versions
+    # don't understand debug symbols for more recent builds
+    set +u
+    export CFLAGS="-fPIC -fPIE ${CFLAGS}"
```

Comment:
The `-fPIE` here is probably unneeded, as is `CXXFLAGS`, but it's done just out of an abundance of precaution.

Reply:
This is no longer be needed since we no longer add `-disable-shared` to our config.